django_app/core/templates/core/base.html

```diff
@@ -96,7 +96,7 @@ <h2 class="govuk-cookie-banner__heading govuk-heading-m">
         {% endblock cookie_banner %}
 
 
-        {% block title %}{% endblock title %}
+        <title>{% block title %}{% endblock title %}Report a suspected breach of trade sanctions - GOV.UK</title>
```

Comment:
Sorry to be a pain but I still think we can squeeze more of the duplicated code into here, otherwise we have to ensure that all of the child `{% block title %}` include spaces after the hyphen, if we do it here there's only 1 place we need to check, if we do it in the kids there will be loads

```suggestion
        <title>{% block title %}{% endblock title %}Report a suspected breach of trade sanctions - GOV.UK</title>
```
```suggestion
        <title>{% block title %}{% endblock title %} - Report a suspected breach of trade sanctions - GOV.UK</title>
```